module-types/Cisco/Meraki-MA-PWR-1025WAC.yaml

```diff
@@ -0,0 +1,9 @@
+---
+manufacturer: Cisco
+model: MA-PWR-1025WAC
+part_number: MA-PWR-1025WAC
+comments: '[1025W power supply](https://meraki.cisco.com/product/switches/switches-accessories/switches-accessories-power/1025-watt-power-supply/)'
```

Comment:
Wouldn't be the text for this link better as "Documentation" ?

Reply:
Or even:
```markdown
1025W Power Supply for MS250-48FP
[Documentation](https://meraki.cisco.com/product/switches/switches-accessories/switches-accessories-power/1025-watt-power-supply/)
```

Reply:
It isn't just documentation.  That is the accessory product page.  It probably could be more descriptive however

Reply:
I would do the same as the switches.

"1025W power supply overview and specification"

Reply:
it's unfortunately the only piece of information that I could find for this at all.
there doesn't seem to be any published specs at all for the PSUs, besides the provided wattage.
the supported models list isn't exhaustive either, as for example the 250W PSUs are explicitly mentioned for MS250-24-HW and MS250-48-HW in https://meraki.cisco.com/lib/pdf/meraki_datasheet_ms250.pdf, yet the product page only lists MX250 and MX450.
what do you think about a more generic text like `1025W power supply product information`?
the `overview and specifications` part comes from the title of the associated page for the switches, which the PSUs don't really have.

Reply:
like this?